packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -535,9 +535,8 @@ class BouncingScrollPhysics extends ScrollPhysics {
   double get minFlingVelocity => kMinFlingVelocity * 2.0;
 
   // Methodology:
-  // 1- Use https://github.com/flutter/scroll_overlay to test with Flutter and
-  //    platform scroll views superimposed.
-  // 2- Record incoming speed and make rapid flings in the test app.
+  // 1- Use https://github.com/flutter/platform_tests/tree/master/scroll_overlay to test with
```

Comment:
You missed updating the rest of the numbered list :)

Reply:
Fixed, accidentally removed point 2